### PR TITLE
[stable/redis] Set metrics annotations by default

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@bitnami.com>

#### What this PR does / why we need it:
Currently, set metrics.enabled=true will have no effect unless the annotations are set by default
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/redis/values-production.yaml [hunk 1]
```diff
@@ -296,8 +296,8 @@ metrics:
     ## otherwise leave blank
     # loadBalancerIP:
     annotations: {}
```
Comment (carrodher) [inline stable/redis/values-production.yaml]: Please, remove `{}`

Comment (juan131) [reply stable/redis/values-production.yaml]: +1

**Pull request merged**